kontan-server/src/blocks.ts

```diff
@@ -100,6 +103,13 @@ export const registerModal: ModalView = {
     {
       type: 'divider',
     },
+    {
+      type: 'header',
+      text: {
```

Comment:
You've done the selector already, can't we have it in the signup modal?
So we can send what office a user selects from the start, and set that in DB on user creation